src/ape/api/transactions.py

```diff
@@ -147,18 +147,11 @@ class ReceiptAPI(BaseInterfaceModel):
 
     contract_address: Optional[str] = None
     block_number: int
-    data: bytes = b""
-    gas_limit: int
-    gas_price: int
     gas_used: int
     logs: List[dict] = []
-    nonce: Optional[int] = None
-    receiver: str
-    required_confirmations: int = 0
-    sender: str
     status: int
     txn_hash: str
-    value: int = 0
+    raw_transaction: TransactionAPI
```

Comment:
Wonder if it should just be `.transaction`, adding raw seems redundant

Reply:
[bc1d0d7](https://github.com/ApeWorX/ape/pull/1015/commits/bc1d0d74f8c9227c80f88b2c923fcf2971a59b04)